### DISCUSSION TITLE
Want to contribute to sidekick project flutter stable version is unknown

### DISCUSSION THREAD

**Discussion by charles0122** [leoafarias/sidekick#321] [Q&A]
I saw that the submission history was changed to stable version three years ago to support versions 2.8 and above, but the current version is 3.22.3, and it seems that this version is not yet supported for development.

**Comment by aguilaair** [comment]
I'm not sure I understand what you mean...
<img width="793" alt="imagen" src="https://github.com/user-attachments/assets/7ff65ada-35b7-4e78-8d6a-2f859a1b51ac">

**Reply by charles0122** [reply]
maybe you stable version is 3.22.2, and i am 3.22.3

Both versions of dart sdk are greater than 3.4, but the project's pubspec.yml file is
```yml
environment:
  sdk: ">=2.16.0 <3.0.0"
```

I use 3.22.3 to run and get an error: intl needs version 0.19.0, the current version is 0.18.1

**Reply by charles0122** [reply]
ok, you dart sdk are greater than 3.4,that not meet the project constraints of >=2.16.0 <3.0.0

**Comment by charles0122** [comment]
why source code version is 0.2.0, but release version is 1.1.1?
